### PR TITLE
Update the Danish translation

### PR DESCRIPTION
**Pull request by Stisen1**
mostly skill naming

### CODE REVIEW

**Comment by Stisen1** [comment]
@zenseii 

Done. I thought this title would be to vague. But of course the reviewers must have a certain repetoire of pull request titles, in order to get as good of an overview as possible. 

Furthermore I will try to make lesser line-changes to the dk.po file in the future, so that it doesn't feel like too much of a chore to get through from time to time. 

Maybe I should go for 30-50 lines at a time?

File: files/lang/dk.po [hunk 1]
```diff
@@ -1392,230 +1392,231 @@ msgid "\"The Dwarfbane!!!!, run for your lives.\""
...
+msgstr "Fizbin-medaljen"
 
 msgid "campaignBonus|Foremost Scroll"
-msgstr ""
+msgstr "Yppersteskriftrullen"
 
 #, fuzzy
 msgid "campaignBonus|Gauntlets"
-msgstr "Avanceret Ørneblik"
+msgstr "Ballisten"
```
[the remark this answers is not in the document]
Comment (Stisen1) [reply files/lang/dk.po:1451]: I have no idea why I wrote "ballisten" there!... I think I have done what you asked me to do now. My local directory has been updated to this date, and I can see a list of your changes in the "history" channel. 

I have copied a version of this new file into another directory outside "fheroes2", so I finally can start editing my incomplete mess of a translation :D

[the remark this answers is not in the document]
Comment (Stisen1) [reply files/lang/dk.po]: Hi, @zenseii 

I had a crying baby and an angry wife by the time of my request process. I believe you should be able to see a pull request called "Update danish translation" accompanied with explainational comments just about now.

It really makes my work a lot easier, now that I can save my po-edits as a mo-file and test it directly inside the game itself. There's a lot of issues with the font-sizes and misunderstandings with what contexts the translations pop-up in, that I need to figure out and explore as I go on with the translation process. It just got way more fun :-D

**Review by ihhub — CHANGES_REQUESTED** [review]
Hi @Stisen1 , I left 2 comments in this pull request. Could you please check them?

**Review by ihhub — APPROVED** [review]

**Comment by ihhub** [comment]
@Stisen1 , thanks a lot for this update!

**Pull request merged**